crm_rma_stock_location/__init__.py

```diff
@@ -19,5 +21,14 @@
 #
 ##############################################################################
 
-from . import stock_warehouse
-from . import product
+from . import models
+from . import wizards
+from openerp import SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
```

Comment:
FYI I'm talking about of a refactory similar to this one:
https://github.com/OCA/product-attribute/blob/9.0/product_pricelist_fixed_price/__init__.py#L19